src/core/components/providers/markdown.jsx

```diff
@@ -22,7 +23,15 @@ function Markdown({ source, className = "" }) {
         linkify: true,
         linkTarget: "_blank"
     }).render(source)
-    const sanitized = sanitizer(html)
+    var sanitized = sanitizer(html)
+
+    //Create deepLinks
+    sanitized = manipHtmlStr(sanitized, 'a', function(elem) {
```

Comment:
Avoid reassigning to the same variable here, create a new one (`manipulated` maybe?) and make both variables `const`s please 😄